experimental/presentation.md

```diff
@@ -0,0 +1,134 @@
+# Advanced Divina Presentation Hints
+
+The Presentation Hints extension defines a number of hints for User Agents about the way content <strong class="rfc">should</strong> be presented to the user.
+
+## 1. Presentation Hints in `metadata`
+
+In order to provide publication-wide Presentation Hints, this extension introduces a new element in the `metadata` object of [the Readium Web Publication Manifest](https://readium.org/webpub-manifest):
+
+| Key   | Semantics | Type     | 
+| ----- | --------- | -------- |
+| `presentation` | Publication-wide presentation hints. | Object |
+
+The following elements <strong class="rfc">may</strong> be included in `presentation`:
+
+- [`clipped`](#clipped)
+- [`fit`](#fit)
+- [`orientation`](#orientation)
+
+## 2. Presentation Hints as Link Properties
+
+In addition to publication-wide hints, this extension defines a number of Link Properties to provide resource-level hints in `readingOrder` or `resources`.
+
+The following elements <strong class="rfc">may</strong> be included in `properties`:
+
+- [`clipped`](#clipped)
+- [`fit`](#fit)
+- [`orientation`](#orientation)
+
+## 3. Properties
+
+### clipped
+
+The `clipped` property is meant to adapt visual resources to any given viewport ratio. The clipped areas <strong class="rfc">must not</strong> contain information which are mandatory for the comprehension of the resource. 
+
+| Key   | Semantics | Type     | Values    | Default |
+| ----- | --------- | -------- | --------- | ------- |
+| `clipped` | Specifies whether or not the parts of a linked resource that flow out of the viewport are clipped.  | Boolean  | `true` or `false` | `false` |
+
+*In this example, all resources are scaled to fit the viewport height and clipped to fit different viewport widths. It behaves like turbomedia.*
+
+```json
+"metadata": {
+  "readingProgression": "ttb",
```

Comment:
I guess this example should use `layout=scrolled` instead.

Reply:
Correct, I'll fix it.